### PR TITLE
Mythos Mod - Start of conversion, remove zombies from zombie game!

### PR DESCRIPTION
**Pull request by ghost**
#### Summary
Mods "Mythos Mod - Removes zombies from game"

#### Purpose of change
Removes all the zombies from the game - a very work-in-progress start to a more Lovecraftian-focused version of Cataclysm set in the 1920s, with 99.9% of the population mysteriously disappearing at midnight of day 0.  Players will take on the roles of investigators in a variety of professions battling the denizens of the shadows to recover and rebuild society.

#### Describe the solution
 - Add zombie species to monster blacklist
 - Add fungus zombie, child fungus zombie, fungus gasbag, fungus smoker, fungus boomer, fungus skeleton,  fungus skeleton brute and fungus skeleton hulk to monster blacklist

#### Describe alternatives you've considered
Not removing the zombies and making the game empty, however first step towards the conversion 

#### Testing
Loaded mod into base game, load correctly

#### Additional context
zombies - very quiet

### CODE REVIEW

**Comment by curstwist** [comment]
Great time period for a mod, looking forward go seeing how this goes.

**Comment by SStelioss** [comment]
Sounds like a splendid idea. Can't wait for this

**Comment by RougeNPS** [comment]
Honestly i think this is a great idea.

But the mod needs a more Lovecraftian name to fit.

**Comment by occulti** [comment]
> Honestly i think this is a great idea.
> 
> But the mod needs a more Lovecraftian name to fit.

The name is one I have been fond of for years but you might be right, a name is everything - don't want to be as on the nose as calling it Cthulhu but I think Mythos has a nice ring to it...

**Comment by Theawesomeboophis** [comment]
Maybe something like Lovecraft Days Ahead, but that may be too on the nose or weird sounding. Whatever its named, it sounds like an interesting idea and I am excited for it.

**Comment by John-Candlebury** [comment]
If you do want a name suggestion, mine is:  `The silver gate`, its referenced in Lovecraft's dreamland stories (and thus its part of the mythos but relatively obscure), where its both `the gate of dreams`  and an analogy for transcendent knowledge guarded by  Yog-Sothoth.

**Comment by ifffrt** [comment]
This is nice. If only I was able to get up my ass and get my Crazy Cataclysm mod ideas to work.

**Comment by TealcOneill** [comment]
Occulti sorry for being a tad off topic I just don't know any other way to contact you. In the contributing.md there's a list of tags you can use to automatically close PRs. Recently several of your PRs lacked this which left the issues open. It would be great if you could try and use those in the future.

**Comment by occulti** [comment]
> Occulti sorry for being a tad off topic I just don't know any other way to contact you. In the contributing.md there's a list of tags you can use to automatically close PRs. Recently several of your PRs lacked this which left the issues open. It would be great if you could try and use those in the future.

No apology needed, I will endeavour to use these in future - I thought I was using the fixes\closes XXXX correctly but I will re-read the docs

**Comment by occulti** [comment]
ping @Maleclypse - have tried changing the JSON in the same way you did to get the tests to do something but no luck, anything else I can try please?

**Comment by Maleclypse** [comment]
Super weird I’ll try and run the tests can you reach out to GitHub support and show them a screenshot of what’s happening and see if they have an idea?

File: data/mods/Mythos/modinfo.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "type": "MOD_INFO",
+    "id": "Mythos",
+    "name": "[WIP] Mythos Mod - No Zombies",
+    "authors": [ "Occulti" ],
+    "description": "Removes all the zombies (including fungus zombies) from the game - a very work-in-progress start to a more Lovecraftian-focused version of Cataclysm set in the 1920s, with 99.9% of the population mysteriously disappearing at midnight of day 0.  Players will take on the roles of investigators in a variety of professions battling the denizens of the shadows to recover and rebuild society.",
+    "category": "content",
+    "dependencies": [ "dda" ],
+    "version": "0.1"
```
Comment (Maleclypse) [inline data/mods/Mythos/modinfo.json:10]:
```suggestion
    "version": "0.1a”
```

**Comment by occulti** [comment]
> Super weird I’ll try and run the tests can you reach out to GitHub support and show them a screenshot of what’s happening and see if they have an idea?

Thanks, will do

**Comment by occulti** [comment]
@Maleclypse I have logged a support ticket as my actions are currently disabled so it may be that - interestingly I just altered the JSON as it was failing and that has caused the tests to not run again so it is definitely directly related to my account - hopefully resolved soon but could you edit again to kick the tests off please?

**Comment by occulti** [comment]
Closed due to resubmission by another user due to issues with actions and automatic checks

**Pull request closed without merging**